WeatherApp/WeatherApp/Feature/CityScreen/CityScreenView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+
+struct CityScreenView: View {
+
+    @ObservedObject var viewModel: CityScreenViewModel
+
+    let columns = [
+        GridItem(.flexible(), spacing: 18),
+        GridItem(.flexible())]
+
+    var body: some View {
+        ZStack {
+            Color.black
+                .ignoresSafeArea()
+
+            if let weather = viewModel.weather {
+                ScrollView {
+                    VStack {
+
+                        NavBar(backAction: viewModel.goBack)
```

Comment:
usually navigation bar is not in the scroll view but fixed to the top. You'll have to add a VStack that will contain navbar and ScrollView